apps/funke/tsconfig.json

```diff
@@ -1,3 +1,9 @@
 {
-	"extends": "../../tsconfig.json"
+	"extends": "../../tsconfig.json",
+	"compilerOptions": {
+		"rootDir": "./",
+		"paths": {
+			"@/*": ["./apps/funke/*"]
```

Comment:
I am not really fan of the `@/` style. Can we just do `@app/*`? Makes way more sense to me as `@/` is quite vague.